h/api/storage.py

```diff
@@ -140,3 +146,7 @@ def _prepare(annotation):
     transform.normalize_annotation_target_uris(annotation)
 
     transform.add_nipsa(annotation)
+
+
+def _postgres_enabled():
+    return pyramid.threadlocal.get_current_request().feature('postgres')
```

Comment:
I agree. According to @nickstenning, it's bad but okay in this case as it will only be in the code base for a short amount of time. As soon as we migrated and switched over the production service, this will be removed (along with a lot of other code).